department-of-veterans-affairs/vets-website
# Remove Claim Estimate from Complete phase

Claim estimates are currently inaccurate and were confusing our users.
    In order to reduce this confusion we will remove the claim estimate
    field from the Complete phase. This also removes the outdated COVID-19
    message when unable to provide an estimate.


## Summary

- Removed ClaimEstimate render logic and components

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#51579

## Testing done

- Previously: Estimate field or COVID message shown in Completed step
- Navigate to /track-claims as logged in user with various claims (228)
  - View details of a claim that is not yet completed
  - Ensure there is no estimated date or COVID message in Completed step


## Screenshots
_Note: This field is mandatory for component work and UI changes (non-component work should NOT have screenshots)._

| | Before | After |
| --- | --- | --- |
| Mobile | | |
| Desktop |
![image](https://user-images.githubusercontent.com/462039/211650552-4045bd27-ef3f-4fa6-a65b-9d78f8c318c3.png)
 |![image](https://user-images.githubusercontent.com/462039/211648931-4d3f2b6f-ff99-4393-a4ad-84fc220f1fa4.png)|

## What areas of the site does it impact?
Claim Status Tool

## Acceptance criteria


## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
